chaco/tools/better_selecting_zoom.py

```diff
@@ -38,14 +37,6 @@ class BetterSelectingZoom(AbstractOverlay, BetterZoom):
     #: the tool to actually take effect.
     minimum_screen_delta = Int(10)
 
-    # -------------------------------------------------------------------------
-    # deprecated interaction controls, used for API compatability with
-    # SimpleZoom
-    # -------------------------------------------------------------------------
-
-    #: Conversion ratio from wheel steps to zoom factors.
-    wheel_zoom_step = Property(Float, observe="zoom_factor")
-
     #: The key press to enter zoom mode, if **always_on** is False.  Has no effect
     #: if **always_on** is True.
     enter_zoom_key = Instance(KeySpec, args=("z",))
```

Comment:
Aren't `enter_zoom_key` and `exit_zoom_key` also part of the deprecated interface? I see those traits on the `SimpleZoom` class. Sadly, they weren't explicitly marked as deprecated.

But then again, these traits do feel like they belong to this class/api.

Reply:
Yeah these weren't throwing warnings, and they also appear to be being explicitly used by the class so I figured they would need to stay